napalm_iosxr/iosxr.py

```diff
@@ -1784,3 +1784,57 @@ def get_config(self, retrieve='all'):
                 self.device._execute_config_show('show configuration merge'))
 
         return config
+
+    def get_isis_hostnames(self):
```

Comment:
Methods that are not part of the public API (available via napalm-base) should start with an underscore so this should be `_ get_isis_hostnames`

Reply:
Sure @dbarrosop , can modify the name

Does it make sense to include get_isis_hostname as part of the public API ?

Reply:
Do you think the method will be useful by itself or do you think people might always prefer to use `get_isis_adjacencies`?

Reply:
Good question, the reason I say this is because XR has a similar cli "show isis hostnames" which gives the mapping with the ISIS level.

My reason to add get_isis_hostnames as part of get_isis_adjacencies was to make it more easy to identify the router names as by default XR spits out the system-id in XML while in the CLI output it was the hostname.

I will make it part of the public API if you guys are okay

Reply:
Sounds good to me.

Reply:
Have added the base method for get_isis_hostnames